### ISSUE TITLE
from_file method of tokenizers.Tokenizer does not exist

### ISSUE THREAD

**Issue by llStringll** [huggingface/tokenizers#341]
I trained a Byte-level BPE tokenizer on reddit pushshift, and I saved it like so:
```
tok=tokenizers.ByteLevelBPETokenizer()
tok.train(['my_file_here.txt'])
tok.save("my_dir_here")
```
This worked as expected, saved vocab.json and merges.txt

Now,  as shown in tokenizer repo's readme for python, to load a saved tokenizer one can do so:
```
tokenizer = Tokenizer.from_file("byte-level-bpe.tokenizer.json")
```
It throws an error stating that Tokenizer doesn't have an attribute "from_file"
Well, now how can one load a trained tokenizer?

PS- Can one just parse vocab.json into newline-separated tokens similar to vocab.txt of transformers.PreTrainedTokenizers and load it using transformers.PreTrainedTokenizer.from_pretrained() ?

**Comment by n1t0** [comment]
Hi @llStringll. I don't know which version of `tokenizers` you are using, but I think you are using an old one.
In `0.8.0+` the `save` method now saves an entire tokenizer json file, as opposed to the `vocab.json` and `merges.txt` files generated with all the previous versions (These are the files generated for BPE tokenizers). So if you have a `json` file, you should be able to load it with `Tokenizer.from_file` (available from `0.8.0+` too), otherwise you can keep loading the old way: `ByteLevelBPETokenizer(vocab_file, merges_file)`.

In `transformers`, you should be able to load a tokenizer using the `vocab.json` and `merges.txt` files too, at least for tokenizers that use these files. The `vocab.txt` file you are talking about is for `BERT` like models, using a `WordPiece` tokenizer, and not a `BPE`.

**Comment by llStringll** [comment]
Oh, thank you for clarifying, especially, that fact about vocab.txt being for only WordPiece not for BPE, i see this now.
